src/OpenRoad.cc

```diff
@@ -438,28 +438,39 @@ void OpenRoad::writeCdl(const char* outFilename,
 }
 
 void OpenRoad::readDb(const char* filename)
+{
+  std::ifstream stream;
+  stream.open(filename, std::ios::binary);
```

Comment:
configure stream to throw exception, missing...

Reply:
stream.exceptions(std::ifstream::failbit | std::ifstream::badbit
                    | std::ios::eofbit);

Reply:
The code was just moved - it is already handled.